applications/GeoMechanicsApplication/custom_constitutive/README.md

```diff
@@ -0,0 +1,28 @@
+# Constitutive laws
+
+
+## Incremental linear elastic interface law
+The constitutive law for an interface element relates tractions $\tau$ to relative displacement $\Delta u$.
+Relative displacement for interface element is the differential motion between the two sides of the interface. As a
+consequence the relative displacement has unit of length [$\mathrm{L}$] and the stiffness has unit of force over cubic length [$\mathrm{F/L^3}$].
```

Comment:
These mathrms don't render correctly, I think reversing dollar and `[` should work `$[\mathrm{F/L^3}]$` (would look like $[\mathrm{F/L^3}]$), adding a space between `[` and `$` also works

Reply:
Good catch. Done.